README.md

```diff
@@ -32,6 +32,12 @@ $ python get-pip.py
 $ pip install umpire
 ```
 
```

Comment:
I don't think this is the best place to indicate a known issue.  Perhaps a known issue section might better.  I think the explanation for this error needs to be expanded as the crash is related to added security to restrict multithreading in Mac OS High Sierra and higher in python 2.  Issue is not specific to umpire itself mind you but appears in umpire because it is multithreaded.  If you want I can write something up to this effect.

Reply:
That will be great. I just throw the idea here.